docs/quickstarts/insights-image-builder/insights-image-builder.yml

```diff
@@ -12,5 +12,5 @@ spec:
   icon: data:image/svg+xml;base64,PHN2ZyBpZD0iZTA1M2ZiYWQtY2I0Ny00OGNiLThiNDMtZDczY2RmMzQyNGJjIiBkYXRhLW5hbWU9IkxheWVyIDEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDM3IDM3Ij4KICA8ZGVmcz4KICAgIDxzdHlsZT4KICAgICAgLmE0NjBjMWU3LWMyNjAtNDc5MS1iZjc0LTI0ZmE4OGU4MTcwZSB7CiAgICAgICAgZmlsbDogI2UwMDsKICAgICAgfQoKICAgICAgLmU4YTQ2MmQ1LTgzYWUtNGRjMi04OTVlLWE3NDU3ZjY2MDNjYSB7CiAgICAgICAgZmlsbDogI2ZmZjsKICAgICAgfQogICAgPC9zdHlsZT4KICA8L2RlZnM+CiAgPHBhdGggZD0iTTI3LjUuNUg5LjVhOSw5LDAsMCwwLTksOXYxOGE5LDksMCwwLDAsOSw5aDE4YTksOSwwLDAsMCw5LTlWOS41YTksOSwwLDAsMC05LTlaIi8+CiAgPHBhdGggY2xhc3M9ImE0NjBjMWU3LWMyNjAtNDc5MS1iZjc0LTI0ZmE4OGU4MTcwZSIgZD0iTTI2LjUsMjcuMTI1aC02YS42MjUuNjI1LDAsMCwxLS42MjUtLjYyNXYtNmEuNjI1LjYyNSwwLDAsMSwuNjI1LS42MjVoNmEuNjI1LjYyNSwwLDAsMSwuNjI1LjYyNXY2QS42MjUuNjI1LDAsMCwxLDI2LjUsMjcuMTI1Wm0tNS4zNzUtMS4yNWg0Ljc1di00Ljc1aC00Ljc1WiIvPgogIDxwYXRoIGNsYXNzPSJhNDYwYzFlNy1jMjYwLTQ3OTEtYmY3NC0yNGZhODhlODE3MGUiIGQ9Ik0yNi41LDE3LjEyNWgtNmEuNjI1LjYyNSwwLDAsMS0uNjI1LS42MjV2LTZhLjYyNS42MjUsMCwwLDEsLjYyNS0uNjI1aDZhLjYyNS42MjUsMCwwLDEsLjYyNS42MjV2NkEuNjI1LjYyNSwwLDAsMSwyNi41LDE3LjEyNVptLTUuMzc1LTEuMjVoNC43NXYtNC43NWgtNC43NVoiLz4KICA8cGF0aCBjbGFzcz0iYTQ2MGMxZTctYzI2MC00NzkxLWJmNzQtMjRmYTg4ZTgxNzBlIiBkPSJNMTYuNSwxNy4xMjVoLTZhLjYyNS42MjUsMCwwLDEtLjYyNS0uNjI1di02YS42MjUuNjI1LDAsMCwxLC42MjUtLjYyNWg2YS42MjUuNjI1LDAsMCwxLC42MjUuNjI1djZBLjYyNS42MjUsMCwwLDEsMTYuNSwxNy4xMjVabS01LjM3NS0xLjI1aDQuNzV2LTQuNzVoLTQuNzVaIi8+CiAgPHBhdGggY2xhc3M9ImU4YTQ2MmQ1LTgzYWUtNGRjMi04OTVlLWE3NDU3ZjY2MDNjYSIgZD0iTTE2LjUsMTkuODc1aC01YS42MjUuNjI1LDAsMCwwLDAsMS4yNWgzLjQ5MWwtNC45MzMsNC45MzNhLjYyNS42MjUsMCwwLDAsLjg4NC44ODRsNC45MzMtNC45MzNWMjUuNWEuNjI1LjYyNSwwLDAsMCwxLjI1LDB2LTVBLjYyNS42MjUsMCwwLDAsMTYuNSwxOS44NzVaIi8+Cjwvc3ZnPgo=
   description: Create customized images and upload images to target cloud environments such as Amazon Web Services or Google Cloud Platform.
   link:
-    href: https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html/creating_customized_images_by_using_insights_image_builder
+    href: https://docs.redhat.com/en/documentation/red_hat_lightspeed/1-latest/html/creating_customized_images_by_using_insights_image_builder
```

Comment:
This is one of Sasha's titles. It should say "red_hat_lightspeed_image_builder", but better verify that with her.